airflow/providers/amazon/aws/hooks/logs.py

```diff
@@ -63,7 +63,7 @@ def get_log_events(
         start_time: int = 0,
         skip: int = 0,
         start_from_head: bool = True,
-        continuation_token: ContinuationToken = ContinuationToken(),
+        continuation_token: ContinuationToken | None = None,
```

Comment:
with the previous version, I didn't realize that the "default object" is created only once. And since we modify it in the method, the default object is modified, and when this method is called again with the default parameters, instead of getting a clean object, we get the modified one from the last call.